modules/apps/collaboration/document-library/document-library-api/src/main/java/com/liferay/document/library/configuration/DLConfiguration.java

```diff
@@ -26,8 +27,12 @@
 	id = "com.liferay.document.library.configuration.DLConfiguration",
 	localization = "content/Language", name = "dl.configuration.name"
 )
+@ProviderType
```

Comment:
@jonathanmccann that's so much nicer :)  I forgot we could do this on our configuration files.